File: lib/models/partida.dart
class Partida {
  final int id;
  final int idDeporte;
  final int idCreador;
  final DateTime fecha;
  final String hora;
  final String lugar;
  final String descripcion;
  final int cantJugadores;
  String estado;

  Partida({
    required this.id,
    required this.idDeporte,
    required this.idCreador,
    required this.fecha,
    required this.hora,
    required this.lugar,
    required this.descripcion,
    required this.cantJugadores,
    required this.estado,
  });

}

File: backend/models/partida.py
from sqlalchemy import Column
from sqlalchemy import Integer
from sqlalchemy import String
from sqlalchemy import Date
from sqlalchemy import Time


from sqlalchemy.orm import declarative_base


Base = declarative_base()

class Partida(Base):
    __tablename__ = "partidas"

    id_partida = Column(
        Integer,
        primary_key=True)
    id_creador = Column(Integer)
    id_deporte = Column(Integer)
    fecha = Column(Date)
    hora = Column(Time)
    cant_jugadores = Column(Integer)
    lugar = Column(String(200))
    id_ubicacion = Column(Integer)
    descripcion = Column(String(500))
    estado = Column(String(20))